<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="main">
     <property name="styleSheet">
      <string notr="true">QFrame#main{
	background: #121212;
	border-radius: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="title">
        <property name="styleSheet">
         <string notr="true">QLabel{
	color: #c91c42;
}

QPushButton{
	image: url(:/icons/svg/x.svg);
	background: rgba(1, 1, 1, 0);
	color: rgba(255, 255, 255, 0.7)
}

QPushButton:hover{
	background: rgba(255, 255, 255, 0.03);
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_icon">
           <property name="font">
            <font>
             <family>Malgun Gothic</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>ICON</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Close</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="choice">
        <property name="styleSheet">
         <string notr="true">QLineEdit, QToolButton, QRadioButton{
	color: rgba(255, 255, 255, 0.7);
}

QLabel{
	color: rgba(255, 255, 255, 0.5);
	background: rgba(0, 0, 0, 0)
}

QLineEdit{
	background: rgba(255, 255, 255, 0.05);
	border: 0px;
	border-radius: 3px;
}

QToolButton{
	border: 0px;
	border-radius: 3px;
}

QToolButton:hover, QRadioButton:indicator:hover{
	background: rgba(255, 255, 255, 0.1);
}

QRadioButton:indicator{
	width: 13px;
	height: 13px;
	image: url(:/icons/svg/circle.svg);
}

QRadioButton:indicator:checked{
	with: 13px;
	height: 13px;
	image: url(:/icons/svg/disc.svg);
}

QLineEdit:disabled{
	background: rgba(255, 255, 255, 0.02);
	color: rgba(255, 255, 255, 0.3);
}

QToolButton:disabled{
	color: rgba(255, 255, 255, 0.5);
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_radio">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButton_default">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Default</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_none">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_option">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Option</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_path">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_path">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="DropLineEdit" name="lineEdit_path">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_path">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QFrame" name="button">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	width: 50px;
	height: 20px;
	color: #0f0e0e;
	background: rgba(201, 28, 66, 0.7);
	border: 0px;
	border-radius: 3px;
}

QPushButton:hover{
	background: rgba(201, 28, 66, 0.9)
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item alignment="Qt::AlignBottom">
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignBottom">
       <widget class="QFrame" name="size_grip">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customs</header>
   <slots>
    <slot>set_placeholder_text()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../imgs/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_option</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_path</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_option</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButton_path</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_option</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_path</receiver>
   <slot>set_placeholder_text()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
